Resolve 23 ISO comments (#636)
buildingSMART/IFC4.3.x-development: docs/schemas/core/IfcProductExtension/Entities/IfcBridge.md

Author: @pjanck can you have a look at this?

### Product Local Placement

The local placement for IfcBridge is defined in its supertype _IfcProduct_. It is defined by _IfcLocalPlacement_ or by _IfcLinearPlacement_ which defines the local coordinate system that is referenced by all geometric representations.

> * The *PlacementRelTo* relationship of _IfcLocalPlacement_ shall point (if relative placement is used) to the _IfcSpatialStructureElement_ of type _IfcSite_, or of type _IfcBuilding_ (e.g. to position a building relative to a building complex, or a building section to a building).

Reviewer: ```suggestion
* The *PlacementRelTo* relationship of _IfcLocalPlacement_ shall point (if relative placement is used) to the IfcSpatialStructureElement containing the bridge (if available).
```

> * If the relative placement is not used, the absolute placement is defined within the world coordinate system.

Reviewer: ```suggestion
* Linear placement can be used to place the bridge at a specific distance along a (alignment) curve.
* If neither relative placement nor linear placement is, the absolute placement is defined within the world coordinate system by means of a local placement PlacementRelTo set to be undefined.
```

#### IfcProject

Direct assignment to project, if the bridge is the outermost spatial container, and no site information is provided for bridge projects.

#### IfcSite

Assignment to site, if the bridge is the spatial container for the bridge project with site information.

#### IfcBridge

Assignment to another bridge as spatial container, e.g. if this bridge represents a bridge section.

### Spatial Container

> NOTE  If there are building elements and/or other elements directly related to the _IfcBridge_, they are associated with the _IfcBridge_ by using the objectified relationship _IfcRelContainedInSpatialStructure_. The _IfcBridge_ references them by its inverse relationship:
> *  _IfcBridge.ContainsElements_ -- referencing any subtype of _IfcProduct_ (with the exception of other spatial structure element) by _IfcRelContainedInSpatialStructure.RelatedElements_.

#### IfcElement

Physical elements that are directly related to the building.

#### IfcAnnotation

Annotations that are directly related to the building.

#### IfcPositioningElement

Positioning elements that are directly related to the building.

> 

Reviewer 2: This seems unrevised after (what I imagine was) copy-paste.
